scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/kafka/protocol.kt

```diff
@@ -0,0 +1,157 @@
+            DataType.UINT32 -> {
+                val v = input.contents.uintContentsList.flatMap { ByteBuffer.allocate(UInt.SIZE_BYTES)
+                    .order(ByteOrder.LITTLE_ENDIAN).putInt(it).array().toList() }.toByteArray()
+                builder.addRawInputContents(v.toByteString())
+            }
+            DataType.UINT64 -> {
+                val v = input.contents.uint64ContentsList.flatMap { ByteBuffer.allocate(ULong.SIZE_BYTES)
+                    .order(ByteOrder.LITTLE_ENDIAN).putLong(it).array().toList() }.toByteArray()
+                builder.addRawInputContents(v.toByteString())
+            }
+            DataType.INT8 -> {
+                val v = input.contents.intContentsList.flatMap { ByteBuffer.allocate(1)
+                    .order(ByteOrder.LITTLE_ENDIAN).put(it.toByte()).array().toList() }.toByteArray()
+                builder.addRawInputContents(v.toByteString())
+            }
+            DataType.INT16 -> {
+                val v = input.contents.intContentsList.flatMap { ByteBuffer.allocate(Short.SIZE_BYTES)
+                    .order(ByteOrder.LITTLE_ENDIAN).putShort(it.toShort()).array().toList() }.toByteArray()
+                builder.addRawInputContents(v.toByteString())
+            }
+            DataType.INT32 -> {
+                val v = input.contents.intContentsList.flatMap { ByteBuffer.allocate(Int.SIZE_BYTES)
+                    .order(ByteOrder.LITTLE_ENDIAN).putInt(it).array().toList() }.toByteArray()
+                builder.addRawInputContents(v.toByteString())
+            }
+            DataType.INT64 -> {
+                val v = input.contents.int64ContentsList.flatMap { ByteBuffer.allocate(Long.SIZE_BYTES)
+                    .order(ByteOrder.LITTLE_ENDIAN).putLong(it).array().toList() }.toByteArray()
+                builder.addRawInputContents(v.toByteString())
+            }
+            DataType.BOOL -> {
+                val v = input.contents.boolContentsList.flatMap { ByteBuffer.allocate(1)
+                    .order(ByteOrder.LITTLE_ENDIAN).put(if (it) {1} else {0}) .array().toList() }.toByteArray()
+            }
+            DataType.FP16, // Unclear if this is correct as will be stored as 4 byte floats
+            DataType.FP32 -> {
+                val v = input.contents.fp32ContentsList.flatMap { ByteBuffer.allocate(Float.SIZE_BYTES)
+                    .order(ByteOrder.LITTLE_ENDIAN).putFloat(it).array().toList() }.toByteArray()
+                builder.addRawInputContents(v.toByteString())
+            }
+            DataType.FP64 -> {
+                val v = input.contents.fp64ContentsList.flatMap { ByteBuffer.allocate(Double.SIZE_BYTES)
+                    .order(ByteOrder.LITTLE_ENDIAN).putDouble(it).array().toList() }.toByteArray()
+                builder.addRawInputContents(v.toByteString())
+            }
+            DataType.BYTES -> {
+                val v = input.contents.bytesContentsList.flatMap { ByteBuffer.allocate(it.size() + 4)
```

Comment:
```suggestion
                val v = input.contents.bytesContentsList.flatMap { ByteBuffer.allocate(it.size() + Int.SIZE_BYTES)
```
:upside_down_face: For consistency with the integer handling